.eslintrc.json

```diff
@@ -3,17 +3,19 @@
     "es6": true,
     "node": true
   },
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint"],
   "parserOptions": {
     "ecmaVersion": 9,
     "sourceType": "module"
   },
   "rules": {
-    "no-unused-vars": [
+    "@typescript-eslint/no-unused-vars": [
       "error",
       { "argsIgnorePattern": "^_", "varsIgnorePattern": "^debug$|^_" }
     ],
     "quote-props": [ "error", "consistent" ],
-    "semi": [ "error", "always" ],
+    "@typescript-eslint/semi": [ "error", "always" ],
```

Comment:
Same: will this still apply to JS?

Reply:
Didn't explicitly test this one, but presumably given the above.